ui/lib/email.js

```diff
@@ -0,0 +1,7 @@
+export const validateEmail = (email) => {
```

Comment:
Is there a way we can do this via `<input type='email' required />`?

Reply:
I tried to do that first, but for some reason it did not verify the input value. So that is why I do this way. I was planning to make an investigate ticket to see why it did not work after we release the ui